### PR TITLE
Feature/sof 4457

### PR DESCRIPTION
**Pull request by bsmith24**

### CODE REVIEW

File: test/pythonml/integration_configuration.yaml [hunk 1]
```diff
@@ -191,6 +192,14 @@ tests:
       - CLS_randomForest
       - POS_plotROC
 
+  Cls_ReadCSV_StandScale_GradientBoostedClassifier_Parity:
```
Comment (AcylSilane) [inline test/pythonml/integration_configuration.yaml:195]: Looks like there's a typo in the final unit here - we're running an ROC plot in this test, not a parity plot (see naming for the other classification test)

File: test/pythonml/integration_configuration.yaml [hunk 2]
```diff
@@ -68,6 +68,7 @@ unit_shortnames:
 
   # Classifiers
   CLS_randomForest: "pyml:model:random_forest_classification:sklearn.pyi"
+  CLS_gradBoost: "pyml:model:gradboosted_classification:sklearn.pyi"
```
Comment (AcylSilane) [inline test/pythonml/integration_configuration.yaml:71]: Check the naming convention with the other gradient boosting algo above. Technically gradient boosting can be performed with any weak learner, not just a decision tree (although sklearn's implementation only allows for trees).

**Review by AcylSilane — COMMENTED** [review]
Overall looks pretty good

File: test/pythonml/integration_configuration.yaml (hunk 1 above)
Comment (bsmith24) [reply test/pythonml/integration_configuration.yaml:195]: Thanks James, I'll rename accordingly.

File: test/pythonml/integration_configuration.yaml (hunk 2 above)
Comment (bsmith24) [reply test/pythonml/integration_configuration.yaml:71]: Ah, got it. Will do. Thanks.

**Comment by timurbazhirov** [comment]
I don't think we need this PR anymore - it can be closed
